### PR TITLE
Adding pep8

### LINKED ISSUE #154: Add pep8
Trollolo has some Python code. We should add [pep8](https://www.python.org/dev/peps/pep-0008), Python standard style checker to the project. It would need to be documented in the [Contributing guide](https://github.com/openSUSE/trollolo/blob/master/CONTRIBUTING.md) and run in Travis.

### PR DESCRIPTION
**Pull request by Ronaq13**
Added pycodestyle (formerly pep8), a Python standard style checker to the project.
It is documented in the Contributing guide and added in travis.yml to run on Travis.

close #154

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,13 +2,15 @@ language: ruby
...
   - gem list bundler
+  - sudo pip install pycodestyle
 rvm:
   - "2.2.3"
   - "2.3.1"
   - "2.4.3"
   - "2.5.0"
 script:
   - 'bundle exec rubocop -D'
+  - 'pycodestyle --first scripts/'
```
Comment (Ana06) [inline .travis.yml]: why using `first` option?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -40,11 +40,16 @@ Install the gem:
 
 `gem install trollolo-<version>.gem`
 
-### To run rubocop
+### To run style checks
```
Comment (Ana06) [inline CONTRIBUTING.md]: checkers

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -40,11 +40,16 @@ Install the gem:
...
 
-To run Rubocop displaying cop names in offense messages:
+* **Ruby**: To run Rubocop displaying cop names in offense messages:
+`bundle exec rake stylecheck:ruby`
 
-`bundle exec rubocop -D`
+* **Python**: We are using pycodestyle (formerly pep8) as a tool to check Python code against some of the style conventions in PEP 8.
```
Comment (Ana06) [inline CONTRIBUTING.md]: For consistent you should explain what is Rubocop

File: .travis.yml (hunk 1 above)
Comment (Ronaq13) [reply .travis.yml]: `first` option shows first occurrence of each error, so we get the overall idea of what are the wrong parts of code while looking at travis logs. It also saves total execution time.

That's why I thought to use it.

File: CONTRIBUTING.md (hunk 2 above)
Comment (Ronaq13) [reply CONTRIBUTING.md]: I will change that.

Comment (Ronaq13) [reply CONTRIBUTING.md]: Done

File: CONTRIBUTING.md (hunk 3 above)
Comment (Ronaq13) [reply CONTRIBUTING.md]: Done

**Comment by Ronaq13** [comment]
Is it good to go now?

File: .travis.yml (hunk 1 above)
Comment (Ana06) [reply .travis.yml]: this is fine for travis, but for the rake task I would suggest rendering everything.

File: setup.cfg [hunk 4]
```diff
@@ -0,0 +1,2 @@
+[pycodestyle]
+max-line-length = 160
```
Comment (Ana06) [inline setup.cfg]: is it too difficult to solve this? :thinking: or is it for consistence with rubocop?

**Comment by Ronaq13** [comment]
@Ana06 is it good to go

**Pull request merged**